node/sc/bridge_manager.go

```diff
@@ -311,7 +315,23 @@ func (bi *BridgeInfo) handleRequestValueTransferEvent(ev *RequestValueTransferEv
 		}
 		logger.Trace("Bridge succeeded to HandleERC20Transfer", "nonce", ev.RequestNonce, "tx", handleTx.Hash().String())
 	case ERC721:
-		handleTx, err = bi.bridge.HandleERC721Transfer(auth, ev.Raw.TxHash, ev.From, ev.To, tokenAddr, ev.ValueOrTokenId, ev.RequestNonce, ev.Raw.BlockNumber, ev.Uri, ev.ExtraData)
+		// get URI of the ERC721
+		var uri string
+		erc721, err := scnft.NewERC721Metadata(ev.TokenAddress, bi.counterpartBackend)
+		if err != nil {
+			logger.Warn("Failed to get erc721 token instance", "erc721", ev.TokenAddress.String(), "onParent", bi.onChildChain)
```

Comment:
if err is not nil, the `uri` has empty string.
is it okay not to return here??
`uri` will be used on line 334

Reply:
Thanks. @laggu :)
I fixed the code as you commented.